src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -421,26 +423,22 @@ export const CreateTransaction = ({
           label="Sending To"
           type="text"
           onChange={(e) => {
-            let newFormData = {
-              ...formData,
-              // isAccountFunded will be updated on blur, this will make sure
-              // we don't have outdated unfunded error still showing.
-              isAccountFunded: true,
-              toAccountId: e.target.value,
-            };
+            // Touched means that the address was modified. If it was changed
+            // back to the initial value, we still want to make all checks
+            // again.
+            setIsAccountIdTouched(true);
+            setIsAccountFunded(true);
+            setToAccountId(e.target.value);
 
             if (federationAddressFetchStatus) {
               setFederationAddressFetchStatus(null);
             }
 
             // Reset memo whenever a new known account is found or previous
-            // address was of known account.
+            // address was a known account.
             if (knownAccounts[e.target.value] || knownAccounts[prevAddress]) {
```

Comment:
Should we reset the memo anytime the address changes? Why do we only reset for if it's a known account?

Reply:
We want to keep generic memo messages and clear only when we know that it was required.